src/main/java/com/sirsirae/NewFunction.java

```diff
@@ -0,0 +1,12 @@
+package com.sirsirae;
+
+public class NewFunction {
+
+	public final String MYVARISCOOL="Thelo";
```

Comment:
Codacy found an issue: [Variable 'MYVARISCOOL' must be private and have accessor methods.](https://app.codacy.com/gh/SirSiraeHQ/OpenSourceDashboard/pullRequest?prid=5785300)